src/store/actions/types.ts

```diff
@@ -1,3 +1,8 @@
-export const FETCH_AGENCY = 'FETCH_AGENCY';
-export const FETCH_SUCCESS = 'FETCH_SUCCESS';
-export const FETCH_ERROR = 'FETCH_ERROR';
\ No newline at end of file
+import { createActionsWithPrefix } from '../../utils/action';
+
+export const AgencyActions = createActionsWithPrefix('AGENCY/', {
```

Comment:
Wordt nu 'AGENCY/FETCH' getoond in redux devtools

Reply:
Had geswitched van agency naar person, dus alleen ff de naam anders denk ik dan is die goed